### ISSUE TITLE
Colonist corpses in corpse disposal aren't moved to graves

### ISSUE THREAD

**Issue by StStep** [AnarkisGaming/ATC-Feedback#150]
It appears that once a colonist corpse gets moved to a corpse disposal zone they will not be moved to a colonist grave until you delete (or move away) the corpse disposal zone. This means colonists only get moved to graves if the grave existed before they died if you have a corpse disposal zone, and I expected the corpse disposal zone to work more like a stockpile.

**Comment by SerialKicked** [comment]
Yeah, I noticed this behavior a while back. It's not *technically* a bug, but it's a bit annoying. 

I'll see if I can fix it without messing with the rest of the corpse disposal logic (which idk right now).
